CMakeLists.txt

```diff
@@ -20,6 +20,23 @@ include(EthToolchains)
 include(EthPolicy)
 eth_policy()
 
+if(EXISTS "${CMAKE_SOURCE_DIR}/deps/fmt/.git")
+  message(STATUS "git submodules seem to be already initialized: nothing to do.")
+else()
+  message(STATUS "git submodules seem not to be initialized: implicitly executing 'git submodule update --init'.")
+  find_package(Git)
+  if(Git_FOUND)
+    execute_process(
+      COMMAND git submodule update --init
+      WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
+      RESULT_VARIABLE result
+    )
+    if(NOT result EQUAL 0)
+        message(FATAL_ERROR "Failed to initialize submodules: 'git submodule update --init' failed.")
+    endif()
+  endif()
```

Comment:
I added an error if git was git was not found. I think it's ok to leave it without the version bump to 3.19 just because of that `COMMAND_ERROR_IS_FATAL ANY` thing.